bindings/nodejs/lib/types/utils/type_aliases.ts

```diff
@@ -3,3 +3,6 @@
 
 export type u16 = number;
 export type u64 = bigint;
+
+// U256 mainly used for token amount
```

Comment:
do we care about this comment?

Reply:
I'd personally remove it, not very useful